<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>two</class>
 <widget class="QDialog" name="two">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>391</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STXihei</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(147, 112, 219);</string>
   </property>
   <property name="title">
    <string>Авторизация</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>221</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 15pt &quot;STXihei&quot;;
    color: white;
    transition: .2s linear;
    background: #9370DB;
 box-shadow: 0 0 0 2px white, 0 0 0 4px #3C82F8;</string>
    </property>
    <property name="text">
     <string>Авторизоваться</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;STXihei&quot;;</string>
         </property>
         <property name="text">
          <string>Имя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;STXihei&quot;;</string>
         </property>
         <property name="text">
          <string>Логин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="loginLineEdit">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;STXihei&quot;;</string>
         </property>
         <property name="text">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Уже есть аккаунт</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
